src/main/java/trackitnus/model/task/Task.java

```diff
@@ -119,8 +124,10 @@ public int hashCode() {
 
     @Override
     public String toString() {
+        String completionStatus = isDone ? "[DONE]" : "[NOT DONE]";
         final StringBuilder builder = new StringBuilder();
         builder.append(getName())
+            .append(completionStatus)
             .append(" Date: ")
             .append(getDate());
```

Comment:
It should be in the form of [DONE] + Name + Date: .... right?

Reply:
Hmm yes I suppose. Will change it